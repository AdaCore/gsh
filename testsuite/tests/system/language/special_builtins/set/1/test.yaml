title: set -f
driver: shell_script
