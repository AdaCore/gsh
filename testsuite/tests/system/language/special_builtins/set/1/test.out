tata-2.txt tata.txt
tata*.txt
tata-2.txt tata.txt
